packages/scss/src/components/_nav-side.scss

```diff
@@ -1,72 +1,226 @@
 .navSide {
-	background: #666666;
-	//font-size: _theme("sizes.small.font-size");
-	height: 100%;
+	background: _component("navSide.primary-palette.color");
+	bottom: 0;
 	left: 0;
 	overflow-x: hidden;
 	overflow-y: auto;
+	padding-top: _component("navSide.padding-top");
 	position: fixed;
-	text-align: center;
-	top: 0;
-	width: _theme("components.navSide.width");
+	top: _component("navSide.offset-top");
+	width: _component("navSide.width");
 	z-index: 2;
+
+	// SCROLLBAR
+	// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
+	&::-webkit-scrollbar-track {
+		background-color: _component("navSide.primary-palette.color");
+	}
+
+	&::-webkit-scrollbar {
+		width: _component("navSide.scrollbar.width");
+	}
+
+	&::-webkit-scrollbar-thumb {
+		background-color: lighten(_component("navSide.primary-palette.color"), 20%);
+		border-radius: _component("navSide.scrollbar.border-radius");
+	}
 }
 
-.navSide-link {
-	color: white;
+// NECESSARY TO AVOID MOVEMENT WHEN THE SCROLLBAR APPEARS
```

Comment:
It's a fixed width wrapper to keep its content from shifting when the scrollbar appear